test/unit/redact-it.test.ts

```diff
@@ -63,6 +65,24 @@ describe("Redact-it - Single configs argument", () => {
     });
   });
 
+  it("should use redactWith string if 'replace' mask is used", async () => {
+    const myData = { ...defaultObject };
+    const replacerFunction: ReplacerFunction = redactIt({
+      fields: ["password"],
+      mask: {
+        type: "replace",
+        redactWith: "[REDACTED]",
+      },
+    });
+
+    const stringResult = JSON.stringify(myData, replacerFunction);
+
+    expect(JSON.parse(stringResult)).to.deep.equal({
+      ...defaultObject,
+      password: "[REDACTED]",
```

Comment:
Just a small suggestion here. This may give a better notion to peers that the string can be any.
```suggestion
        redactWith: "(ommited value)",
      },
    });

    const stringResult = JSON.stringify(myData, replacerFunction);

    expect(JSON.parse(stringResult)).to.deep.equal({
      ...defaultObject,
      password: "(ommited value)",
```